packages/atlas/src/views/studio/StudioWelcomeView/StudioWelcomeView.styles.ts

```diff
@@ -48,24 +95,200 @@ export const StyledSignInIllustrationSVG = styled(SvgSigninIllustration)`
   }
 `
 
+export const StepsContainer = styled.div`
+  display: flex;
+  width: 100%;
+  position: relative;
+  padding: ${sizes(1)} 0;
+  align-items: center;
+  margin: ${sizes(6)} 0;
+  box-shadow: inset 0 1px 0 0 ${cVar('colorBorderMutedAlpha')}, inset 0 -1px 0 0 ${cVar('colorBorderMutedAlpha')};
+  ${media.md} {
+    margin: ${sizes(8)} 0;
+    overflow: hidden;
+    padding: ${sizes(3)};
+
+    ::before,
+    ::after {
+      position: absolute;
+      content: '';
+      background: linear-gradient(
+        90deg,
+        #000 0%,
```

Comment:
Figma is broken 😅. I bet this is simple gradient from black to transparency. @toiletgranny do you think we could just do `background: linear-gradient(90deg, rgba(0,0,0,0) 0%, rgba(0,0,0,1) 100%);`?

Reply:
What is this gradient? 😅 Were is it used?

Reply:
![Zrzut ekranu 2022-10-19 o 15 18 26](https://user-images.githubusercontent.com/13706246/196702150-98f6d0e4-1271-4d8f-959d-6b7741e9c3c8.png)

Reply:
Oh, that's a spectacular gradient, indeed. :D Yeah, we can totally use simpler version if you ask me.